app/helpers/application_helper.rb

```diff
@@ -108,11 +108,11 @@ def map_to_citations_string translations
       "<p>#{cit}</p>"
     end
 
-    citations.join
+    "<h3>Sources</h3>" + citations.join
   end
 
   def habitat_text(habitat)
-    habitat[:change_percentage].blank? ? I18n.t('home.habitat_cover.data_deficient') : habitat[:text].html_safe
+    habitat[:change_percentage] == '-' ? I18n.t('home.habitat_cover.data_deficient') : habitat[:text].html_safe
```

Comment:
@stanleypliu This is now going to show nothing when stats are missing - just checked with Osgur and Charlotte today.

Reply:
Am I understanding correctly that we should just miss out that habitat altogether if stats are missing for it? As in, not show a separate card for it?

Reply:
No sorry, the card will show, but there won't be any text where the percentage would normally be. I've set up the templates so they should do this if the change percentage is blank

Reply:
I've changed the change_percentage to be nil now, and have updated the condition in the partial to reflect this.